lib/mortar/command.rb

```diff
@@ -153,12 +151,21 @@ def build_kubeconfig_from_env
         preferences: {},
         current_context: 'mortar'
       )
+    rescue ArgumentError
+      puts "KUBE_TOKEN env doesn't seem to be valid base64 encoding!"
```

Comment:
something like "doesn't seem to be base64 encoded"

also you can use `signal_usage_error "..."` or `abort "..."` and won't need to exit with error separately.

Reply:
and you could do the option processing in the option block:

```ruby
option '--kube-token`, 'TOKEN', 'Kubernetes API access token', environment_variable: 'KUBE_TOKEN' do |token|
  begin
    Base64.decode64(token).tap do |decoded|
      raise ArgumentError unless Base64.encode64(decoded) == token
    end
  rescue ArgumentError
    signal_usage_error 'kube token does not seem to be base64 encoded'
  end
end
```

i think the same should be done for `KUBE_CA` and `KUBE_SERVER` too to document them in the `--help` output (and add a secondary way to pass them in)

but maybe that's a separate pr.